yurt/fabfile/add.py

```diff
@@ -57,7 +53,18 @@
 
 
 @main.command()
-def remote_server():
+@click.option("--git_repo", default=None, help="Git Repo Link to Yurt project")
+@click.option("--env", default=None, help="Environment name (i.e. 'Development', 'Staging')")
+@click.option("--abbrev_env", default=None, help="Abbreviated env name (i.e. 'dev', 'stage')")
+@click.option("--app_host_ip", default=None, help="Public DNS of App Server")
+@click.option("--db_host_ip", default=None, help="IP of DB Server")
+@click.option("--debug", default=None, help="Runs debug mode (use `True` or `False`)")
+@click.option("--num_gunicorn_workers", default=None, help="Number of Gunicorn workers")
```

Comment:
@rmutter: Added some logic below this line to handle `default` gunicorn settings. The reason I didn't put them in the decorators is because it's currently rigged to where the user is only prompted if the default option is `None` or falsy.